modules/ROOT/pages/database-administration/standard-databases/listing-databases.adoc

```diff
@@ -133,28 +133,28 @@ Not returned by `SHOW HOME DATABASE` or `SHOW DEFAULT DATABASE`.
 | Number of primaries for this database reported as running currently.
 It is the same as the number of rows where `role=primary` and `name=this database`.
 
-The value for composite databases is `NULL` because it does not apply to them.
+The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[]
```

Comment:
```suggestion
The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[This change applies to versions 2025.04 and later.]
```
or if you want to keep the 5.LTS version info as well
```suggestion
The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[This change applies to versions 2025.04 and later and 5.26.5 and later.]
```